giskard_vision/landmark_detection/tests/performance.py

```diff
@@ -74,6 +74,12 @@ class NMEMean(Metric):
     def definition(prediction_result: PredictionResult, marks: np.ndarray):
         return np.nanmean(NMEs.get(prediction_result, marks))
 
+    @staticmethod
+    def rank_data(prediction_result: PredictionResult, marks: np.ndarray):
+        predictions = NMEs.get(prediction_result, marks)
```

Comment:
predictions is misleading. How about simply `nmes = ...`?

Reply:
Could you add this method to all the metrics where it belongs?

Reply:
Not sure I did it well, this seems to require a lot of code duplication and I am wondering whether dimensions are the same everywhere... @rabah-khalek wdyt ?